flink-connector-aws/flink-connector-kinesis/pom.xml

```diff
@@ -137,6 +137,12 @@ under the License.
         </dependency>
 
         <!-- Flink ecosystem -->
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-core</artifactId>
+            <version>${flink.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Explicitly specifying dependency with the correct scope to prevent licensing check failures on unused transitive dependencies (flink-shaded-jackson).
Previously `flink-core` has been transitively consumed through `flink-streaming-java`, resulting in maven marking some of the packages as included in `compile` scope.